PhysicsTools/NanoAOD/python/lowPtElectrons_cff.py

```diff
@@ -0,0 +1,252 @@
+    checkCharge = cms.bool(False),          # True = require RECO and MC objects to have the same charge
+    mcStatus    = cms.vint32(1),            # PYTHIA status code (1 = stable, 2 = shower, 3 = hard scattering)
+    maxDeltaR   = cms.double(0.3),          # Minimum deltaR for the match
+    maxDPtRel   = cms.double(0.5),          # Minimum deltaPt/Pt for the match
+    resolveAmbiguities    = cms.bool(True), # Forbid two RECO objects to match to the same GEN object
+    resolveByMatchQuality = cms.bool(True), # False = just match input in order; True = pick lowest deltaR pair first
+)
+
+from PhysicsTools.NanoAOD.electrons_cff import electronMCTable
+lowPtElectronMCTable = cms.EDProducer(
+    "CandMCMatchTableProducer",
+    src = lowPtElectronTable.src,
+    mcMapDressedLep = cms.InputTag("lowPtElectronsMCMatchForTableAlt"),
+    mcMap = cms.InputTag("lowPtElectronsMCMatchForTable"),
+    mapTauAnc = cms.InputTag("matchingLowPtElecPhoton:hasTauAnc"),
+    objName = lowPtElectronTable.name,
+    objType = electronMCTable.objType,
+    branchName = cms.string("genPart"),
+    docString = cms.string("MC matching to status==1 electrons or photons"),
+    genparticles = cms.InputTag("finalGenParticles"), 
+)
+
+################################################################################
+# Sequences
+################################################################################
+
+lowPtElectronSequence = cms.Sequence(modifiedLowPtElectrons
+                                     +updatedLowPtElectrons
+                                     +lowPtPATElectronID
+                                     +isoForLowPtEle
+                                     +updatedLowPtElectronsWithUserData
+                                     +finalLowPtElectrons)
+lowPtElectronTables = cms.Sequence(lowPtElectronTable)
+lowPtElectronMC = cms.Sequence(
+    particleLevelForMatchingLowPt
+    +tautaggerForMatchingLowPt
+    +matchingLowPtElecPhoton
+    +lowPtElectronsMCMatchForTable
+    +lowPtElectronsMCMatchForTableAlt
+    +lowPtElectronMCTable)
+
+################################################################################
+# Modifiers
+################################################################################
+
+from Configuration.Eras.Modifier_run2_nanoAOD_94XMiniAODv1_cff import run2_nanoAOD_94XMiniAODv1
+from Configuration.Eras.Modifier_run2_nanoAOD_94XMiniAODv2_cff import run2_nanoAOD_94XMiniAODv2
+from Configuration.Eras.Modifier_run2_nanoAOD_106Xv1_cff import run2_nanoAOD_106Xv1
+(run2_nanoAOD_94XMiniAODv1 | run2_nanoAOD_94XMiniAODv2 | run2_nanoAOD_106Xv1).toReplaceWith(lowPtElectronSequence,cms.Sequence())
+(run2_nanoAOD_94XMiniAODv1 | run2_nanoAOD_94XMiniAODv2 | run2_nanoAOD_106Xv1).toReplaceWith(lowPtElectronTables,cms.Sequence())
+(run2_nanoAOD_94XMiniAODv1 | run2_nanoAOD_94XMiniAODv2 | run2_nanoAOD_106Xv1).toReplaceWith(lowPtElectronMC,cms.Sequence())
```

Comment:
this is likely creating the failures

add in the or the following previous nano configurations
`run2_nanoAOD_102Xv1 | run2_nanoAOD_94X2016 | run2_miniAOD_80XLegacy`

remove the imports of line 247/248/249 since are already included in from PhysicsTools.NanoAOD.nano_eras_cff import *

Reply:
Done, thanks.